### ISSUE TITLE
App Store Submission Error

### ISSUE THREAD

**Issue by terryworona** [marcosgriselli/ViewAnimator#75]
Getting the following error when trying to archive. 

```
ERROR ITMS-90056: "This bundle Frameworks/ViewAnimator.framework is invalid. 
The Info.plist file is missing the required key: CFBundleVersion. 
Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion"
```

Using Carthage:

```
github "marcosgriselli/ViewAnimator"
```

**Comment by marcosgriselli** [comment]
Thanks for reporting this @terryworona! Seems to be a known problem with some setups. I followed the steps suggested on Flight-School/AnyCodable#20 on 2f197f3 and released version [3.0.2](https://github.com/marcosgriselli/ViewAnimator/releases/tag/3.0.2).
Let me know if the issue is solved!

**Comment by terryworona** [comment]
Nice that resolved it! Thanks!
